docs/architecture/overview.md

```diff
@@ -10,22 +10,22 @@ Each tier only has knowledge of its own responsibilities, and specific channels
-The consumer tier is the entry point for interacting with the SDK. It handles taking the user input and deciding which aggregator(s) should handle processing the input. There are currently 2 different consumers that can be used in the SDK
+The consumer tier is the entry point for interacting with the SDK. It handles taking the user input and deciding which handler(s) should handle processing the input. There are currently 2 different consumers that can be used in the SDK
 1. A RESTful WebAPI that can be called using HTTP requests.
 2. A command line tool that runs directly on a host machine.
 
 Both consumers access the same components in the tiers below.
 
 ## Business Logic Tier
-The business logic tier functions as the processor, taking in the user input and figuring out how to fulfill it. It is comprised of a set of aggregators that know what type of data they need and how to calculate the result. 
+The business logic tier functions as the processor, taking in the user input and figuring out how to fulfill it. It is comprised of a set of handlers that know what type of data they need and how to calculate the result. 
 
-### Aggregator
-Aggregators have knowledge of the underlying data source interfaces in the data tier. An Aggregator takes in consumer requests, calls the specified data source, and performs any data aggregation required before returning the result to the consumer. Each Aggregator is responsible for handling requests specific to a functionality.
+### Handlers
+Handlers have knowledge of the underlying data source interfaces in the data tier. A Handler takes in consumer requests, calls the specified data source, and performs any data aggregation required before returning the result to the consumer. Each Handler is responsible for handling requests specific to a functionality.
 
-Currently, the SDK provides 2 aggregators - `EmissionsAggregator` and `ForecastAgrgegator` to handle requests for actual carbon emissions and forecasted carbon emissions respectively. The `EmissionsAggregator` handles requests for various carbon emissions information. It can calculate the average carbon emissions over a time period, or the best carbon emissions given a set of locations. It can transform forecasted carbon emissions to suit particular use-cases. It can also just deliver the emissions data points in a standard schema without performing any calculations.
-The `ForecastAggregator` 
+Currently, the SDK provides 2 handlers - `EmissionsHandler` and `ForecastHandler` to handle requests for actual carbon emissions and forecasted carbon emissions respectively. The `EmissionsHandler` handles requests for various carbon emissions information. It can calculate the average carbon emissions over a time period, or the best carbon emissions given a set of locations. It can transform forecasted carbon emissions to suit particular use-cases. It can also just deliver the emissions data points in a standard schema without performing any calculations.
```

Comment:
Same comment as before, is it worth mentioning that there is another handler (the location handler) and what it is used for?